services/horizon/internal/ingest/parallel.go

```diff
@@ -85,9 +104,17 @@ func calculateParallelLedgerBatchSize(rangeSize uint32, batchSizeSuggestion uint
 	return (batchSize / historyCheckpointLedgerInterval) * historyCheckpointLedgerInterval
 }
 
-func (ps *ParallelSystems) ReingestRange(fromLedger, toLedger uint32, batchSizeSuggestion uint32) error {
+func totalRangeSize(ledgerRanges [][2]uint32) uint32 {
+	var sum uint32
+	for _, pair := range ledgerRanges {
+		sum += pair[1] - pair[0]
```

Comment:
Shouldn't we add +1 here? Like [1, 3] size is 3 (1, 2, 3) not 2.
```suggestion
		sum += pair[1] - pair[0] + 1
```